draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -153,36 +169,36 @@ properties as long as the undelying algorithms are secure.
 
 # Introduction {#intro}
 
-Post-quantum (PQ) algorithms offer a redesign of traditional algorithms tailored
+Post-quantum (PQ) algorithms offer new constructions based on problems tailored
 towards resisting attack from a quantum computer. Key Encapsulation Mechanisms
 (KEMs), are a standardized algorithm type that can be used to build protocols in
-lieu of traditional, quantum-vulnerable variants such as Diffie-Hellman (DH)
-based protocols.  Upgrading protocols to use PQ KEMs is a priority for the
-protocol design community, due to the possibility of "harvest now, decrypt
-later" attacks.
+lieu of traditional, quantum-vulnerable variants such as finite field or
+elliptic curve Diffie-Hellman (DH) based protocols. Upgrading key establishment
+protocols to use PQ KEMs is a priority for the protocol design community, due to
+the possibility of "harvest now, decrypt later" attacks.
```

Comment:
Recommend removal of HNDL attacks. Motivation for PQC is not specific to that, but due to quantum threats.

Reply:
The idea of mentioning HNDL is just to explain why the focus on KEMs vs. Authentication.  But I don't feel super strongly.

Reply:
Fine, fine, we can delete it :)